spec/services/save_auction_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe SaveAuction do
+  describe '#perform' do
+    context 'given a valid auction' do
+      it 'saves the auction' do
+        auction = build(:auction)
+        SaveAuction.new(auction).perform
+
+        expect(auction).to be_persisted
+      end
+
+      it 'returns true' do
+        auction = build(:auction)
+        saved = SaveAuction.new(auction).perform
+
+        expect(saved).to eq true
+      end
+    end
+
+    context 'given an invalid auction' do
+      it 'does not save the auction' do
+        auction = build(:auction, title: nil)
+
+        expect { SaveAuction.new(auction).perform }
+          .not_to change { Auction.count }
+      end
+
+      it 'is not persisted' do
```

Comment:
it seems like this test and the one above are testing the same thing -- keep one?